tethys_apps/utilities.py

```diff
@@ -248,21 +253,50 @@ def get_custom_setting(app_package, setting_name):
     Returns:
         CustomSetting: The Custom Setting or None if the TethysApp or CustomSetting cannot be found.
     """
-    from tethys_apps.models import TethysApp, CustomSetting
+    from tethys_apps.models import TethysApp, CustomSettingBase
 
     try:
         app = TethysApp.objects.get(package=app_package)
     except TethysApp.DoesNotExist:
         return None
-
     try:
-        setting = CustomSetting.objects.get(tethys_app=app, name=setting_name)
-    except CustomSetting.DoesNotExist:
+        setting = (
+            CustomSettingBase.objects.filter(tethys_app=app)
+            .select_subclasses()
+            .get(name=setting_name)
+        )
+    except CustomSettingBase.DoesNotExist:
         return None
 
     return setting
 
 
+def get_custom_secret_settings(app_package):
+    """
+    Get the  Custom Secret Settings for a specified TethysApp.
+
+    Args:
+        app_package (str): The name/package of the TethysApp.
+
+    Returns:
+        InheritanceQuerySet: A Inheritance Query Set containing Custom Secret Settings, None if the TethysApp, or an empty Inheritance Query Set if the app does not have any SecretCustomSetting.
+    """
```

Comment:
Doc string needs names updated:

- [ ] `get_secret_custom_settings`
- [ ] "Get the SecretCustomSettings for..."
- [ ] "InheritanceQuerySet: A Inheritance Query Set containing SecretCustomSettings, None ..."

Reply:
- [ ] get_secret_custom_settings
- [ ] "Get the SecretCustomSettings for..."
- [ ] "InheritanceQuerySet: A Inheritance Query Set containing SecretCustomSettings, None ..."